libs/effects-ng/src/lib/directive-effects.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { Component, Injectable } from '@angular/core';
+import { TestBed } from '@angular/core/testing';
+import { createAction, createEffect } from '@ngneat/effects';
+import { tap } from 'rxjs';
+import { ofType } from 'ts-action-operators';
+import { Actions } from './actions';
+import { provideEffectsManager } from './provide-effects-manager';
+import { directiveEffects } from './directive-effects';
+
+const spy = jest.fn();
+const loadTodos = createAction('[Todos] Load Todos');
+
+@Injectable()
+class EffectsOne {
+  loadTodos$ = createEffect((actions) =>
+    actions.pipe(ofType(loadTodos), tap(spy))
+  );
+}
+
+@Injectable()
+class EffectsTwo {}
+
+@Injectable()
+class EffectsThree {}
+
+@Component({
+  template: '',
+  standalone: true,
+  hostDirectives: [
+    directiveEffects(EffectsOne),
```

Comment:
I'd consider it a bad practice to use this function twice, so maybe it doesn't worth putting in our tests.

Reply:
Should I remove the same form `provideEffects`?